kratos/modeler/connectivity_preserve_modeler.cpp

```diff
@@ -144,7 +144,11 @@ void ConnectivityPreserveModeler::CopyCommonData(
 
     // These should be safe for SubModelParts
     rDestinationModelPart.SetProcessInfo( rOriginModelPart.pGetProcessInfo() );
-    rDestinationModelPart.SetProperties( rOriginModelPart.pProperties() );
```

Comment:
@ddiezrod just to be sure we @KratosMultiphysics/technical-committee  understand the change you propose:

before we were ensuring that the container of the properties was the same between the original 
model_part and the new one.






after this change you are telling that the container are not the same, however the new container is populated with pointers to the same properties. Is that so?

-- OLD BEHAVIOUR --
origin_mp.Properties[1][DENSITY]=1

new_mp.Properties[1] has density 1

if i add a new property to either the new or the old, the new property will be seen identical in the two places.

-- NEW PROPOSED BEHAVIOUR --
pointers are initally the same, so if one changes density in one, also the other can feel it.
however, we could add a new property to one of the modelparts (or possibly reassign a property) to one of the two WITHOUT AFFECTING the other one

is that correct?

Reply:
that is correct! @miguelmaso opened a MR that is equivalent to this one, adding a test that explains the new behavior. We can close this and continue the discussion there as I think that one is cleaner.

Reply:
https://github.com/KratosMultiphysics/Kratos/pull/9758